test/controllers/medias_controller_test.rb

```diff
@@ -686,6 +686,22 @@ def setup
     assert_equal url, JSON.parse(@response.body)['data']['title']
     assert_equal Lapis::ErrorCodes::const_get('DUPLICATED'), JSON.parse(@response.body)['data']['error']['code']
   end
+
+  test "should refresh cache even if ID changes" do
+    WebMock.enable!
+    WebMock.disable_net_connect!(allow: [/minio/])
+    Media.stubs(:get_id).returns('foo', 'bar', 'foo', 'bar')
```

Comment:
@caiosba if/when you have the time, would you mind explaining how line 693 works again?

Reply:
Sure! It basically means that each time the stubbed method (in this case, `get_id`) is called during that test, the returned value will be different. So, first time it will be `foo`, second time it will be `bar`.

Reply:
why `.returns('foo', 'bar', 'foo', 'bar')` and not `.returns('foo', 'bar')` ?

Reply:
Because it runs four times during that test, 2 in the controller and 2 in the model. I can show you when we pair and explain.